easybuild/easyconfigs/d/DISCOVAR/DISCOVARdenovo-52488-foss-2015a.eb

```diff
@@ -0,0 +1,22 @@
+name = 'DISCOVARdenovo'
+version = '52488'
+
+homepage = 'http://www.broadinstitute.org/software/discovar/blog/'
+description = """DISCOVAR de novo can generate de novo assemblies for both large and small genomes. It currently does not call variants."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = ['%(namelower)s-%(version)s.tar.gz']
+source_urls = ['ftp://ftp.broadinstitute.org/pub/crd/DiscovarDeNovo/latest_source_code']
+
+dependencies = [
+    ('GMP', '6.0.0a'),
```

Comment:
if this is the reason you're using the `foss` toolchain rather than `intel`, consider specifying this dependency as:

```
    ('GMP', '6.0.0a', '', ('GCC', '4.9.2')),
```

and then use `intel/2015a` as toolchain; since `GCC/4.9.2` is included as an indirect dependency in `intel/2015a`, that'll work